testcases/testcase_inverse_problem.py

```diff
@@ -23,7 +23,7 @@
 PARAMETERS = MappingProxyType(
     {
         # Setup parameters for blood flow model
-        "read_network_option": 1,  # 1: generate hexagonal graph
+        "read_network_option": 2,  # 1: generate hexagonal graph
```

Comment:
Maybe always reset the standard testcases to a standard test case with a hexagonal network. for fast testing.

Reply:
Done!